tests/rptest/services/rw_workload.py

```diff
@@ -0,0 +1,199 @@
+# Copyright 2020 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from ducktape.services.service import Service
+import requests
+from time import sleep
+import sys
+from rptest.util import wait_until
+
+
+class CrushedException(Exception):
+    pass
+
+
+class ConsistencyViolationException(Exception):
+    pass
+
+
+OUTPUT_LOG = "/opt/remote/var/rw.log"
+
+
+class RWWorkload(Service):
+    logs = {"rw_stdout_stderr": {"path": OUTPUT_LOG, "collect_default": True}}
+
+    def __init__(self, context, redpanda):
+        super(RWWorkload, self).__init__(context, num_nodes=1)
+        self._redpanda = redpanda
+        self._is_done = False
+        self._node = None
+
+    def is_alive(self, node):
+        result = node.account.ssh_output(
+            "bash /opt/remote/control/alive.sh rw")
+        result = result.decode("utf-8")
+        return "YES" in result
+
+    def is_ready(self):
+        try:
+            self.remote_ping()
+            return True
+        except requests.exceptions.ConnectionError:
+            return False
+
+    ### Service overrides
+
+    def start_node(self, node, timeout_sec=10):
+        node.account.ssh(
+            "bash /opt/remote/control/start.sh rw \"java -cp /opt/verifiers/verifiers.jar io.vectorized.reads_writes.App\""
```

Comment:
Does this new service reduce the error-prone factor that comes from SSH-ing and bash? We still need to execute commands via ducktape SSH mechanism so I'm also concerned that we're simply introducing another level of complexity.

Reply:
I think so, we can't get away from ssh and running the scripts but unlike KafProducer we don't use loops in bash so we use it to start a service in a background and write its PID. Also the start.sh, alive.sh and stop.sh are generic and I hope we may reuse them in other ducktape services.